### PR TITLE
Add info alert for dependents and duplicates in DAM

### PR DESCRIPTION
**Pull request by fichtnerma**
---

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: COM-102
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
Dependants:

image

Duplicates:

image

</details>

### CODE REVIEW

File: packages/admin/cms-admin/src/dependencies/DependencyList.tsx [hunk 1]
```diff
@@ -204,6 +204,12 @@ export const DependencyList = ({ query, variables }: DependencyListProps) => {
                     return `${row.id}_${row.rootColumnName}_${row.jsonPath}`;
                 }}
             />
+            <Alert title={<FormattedMessage id="comet.dam.file.dependants.info.title" defaultMessage="What are dependants?" />} sx={{ marginTop: 2 }}>
+                <FormattedMessage
+                    id="comet.dam.file.dependants.info.content"
+                    defaultMessage="Dependants are all pages, snippets and content in which a particular asset is used, linked or included. With this list, it´s easy to manage or reorganize the integration of your assets."
```
Comment (nsams) [inline packages/admin/cms-admin/src/dependencies/DependencyList.tsx:210]:
```suggestion
                    defaultMessage="Dependants are all pages, snippets and content in which a particular asset is used, linked or included. With this list, it's easy to manage or reorganize the integration of your assets."
```

File: packages/admin/cms-admin/src/dam/FileForm/Duplicates.tsx [hunk 2]
```diff
@@ -98,6 +98,15 @@ const Duplicates: React.FC<{ fileId: string }> = ({ fileId }) => {
                     </ListItem>
                 )}
             </List>
+            <Alert
+                title={<FormattedMessage id="comet.dam.file.duplicates.info.title" defaultMessage="What are duplicate assets?" />}
+                sx={{ marginTop: 2 }}
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/dam/FileForm/Duplicates.tsx:103]: Please stick to the screen design. There the margin is 20px, here it's only 10

```suggestion
                sx={{ marginTop: 4 }}
```

File: packages/admin/cms-admin/src/dependencies/DependencyList.tsx [hunk 3]
```diff
@@ -204,6 +204,12 @@ export const DependencyList = ({ query, variables }: DependencyListProps) => {
                     return `${row.id}_${row.rootColumnName}_${row.jsonPath}`;
                 }}
             />
+            <Alert title={<FormattedMessage id="comet.dam.file.dependants.info.title" defaultMessage="What are dependants?" />} sx={{ marginTop: 2 }}>
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/dependencies/DependencyList.tsx:207]:
```suggestion
            <Alert title={<FormattedMessage id="comet.dam.file.dependants.info.title" defaultMessage="What are dependants?" />} sx={{ marginTop: 4 }}>
```

**Review by johnnyomair — COMMENTED** [review]
Since we're using American English, please change to depend**e**nts: https://www.merriam-webster.com/grammar/spelling-variants-dependent-vs-dependant

**Comment by johnnyomair** [comment]
Should it be possible to "remove" this info boxes? Maybe by storing the information in the local storage? IMO they are very prominent...

**Comment by thomasdax98** [comment]
> Should it be possible to "remove" this info boxes? Maybe by storing the information in the local storage? IMO they are very prominent...

I don't think that's necessary since they are below the DataGrid so they aren't annoying. They look very prominent on the screenshot because the Grids are empty / almost empty. In a real application this won't be the case.

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**